### ISSUE TITLE
Why there is large difference in mAP value on Raspberry Pi 4 and normal server for the same trained model?

### ISSUE THREAD

**Issue by jaskiratsingh2000** [ultralytics/yolov5#4167]
Hey! I am trying to test the **Accuracy Performance** that is the **mAP** value of yolov5 on Raspberry Pi 4.

I trained the pre-trained yolov5s model with the following parameters on the server and with the value of the weights I got, I tested those on the Raspberry Pi for the Accuracy Performance:

- **data** - coco.yaml
- **weights** - yolov5s.pt
- **batch** - 50
- **epochs** - 300
- **cfg** - yolov5s.yaml

by following this command -> `python3 train.py --data coco.yaml --batch 50 --epochs 300 --weights yolov5s.pt --cfg yolov5s.yaml `

but when I transfer the weights file and test the mAP value on raspberry pi with those weights file I am getting 0.000032 mAP value whereas if I test the same weights file on the server I get the mAP value as 0.68.

The computer is a computer then why there is such a large difference in the mAP value that is Accuracy performance on Raspberry Pi 4?

@glenn-jocher Please let me know if you have any idea about this?
Thanks!

**Comment by jaskiratsingh2000** [comment]
@glenn-jocher @batrlatom Please let je know if you have any idea about this as I have been trying to know this past few months and kinda stuck here so please let me know. Would highly appreciate that. Thanks!!
